stock_card/tests/test_stock_card_negative_stock.py

```diff
@@ -177,6 +189,15 @@ def test_01_do_inouts(self):
                 self.create_sale_order(qty=qty, price=costprice)
 
         card_lines = self.get_stock_valuations()
+
+        df = pd.DataFrame(card_lines)
+        tbl_sc = tabulate(df, headers='keys', tablefmt='psql')
+        _logger.info('Gotten Stock Card \n%s', tbl_sc)
+
+        df = pd.DataFrame(self.inv_ids)
+        tbl_sc = tabulate(df, headers='keys', tablefmt='psql')
+        _logger.info('Expected Stock Card \n%s', tbl_sc)
```

Comment:
@zaoral It is intended that the message be printed regardless of the failure happening.

Regards.